torax/sources/runtime_params.py

```diff
@@ -104,3 +120,5 @@ class DynamicRuntimeParams:
   mode: int
   is_explicit: bool
   formula: formula_config.DynamicFormula
+  # Prescribed values on the face grid.
+  prescribed_values: chex.Array
```

Comment:
For most places in TORAX, if they don't have "face" in the var name, it refers to the cell grid. For this variable, can you rename it to `prescribed_face_values` or `prescribed_values_face_grid` or something along those lines?

Reply:
Cool, that's good to know generally!

Reply:
Since prescribed_values uses the TimeInterpolated patterns, then on the config level it's not necessarily on either the face or cell grid, it's on a user-provided grid that then gets interpolated onto the TORAX grids. So the issue I think is actually with the comment rather than the variable name

Reply:
Comment removed in a50e64bded6af7913c40a13bb9179a2e58c2306b

Reply:
Ahh, great point. Thanks for catching that Jonathan.